### PR TITLE
Simplify if-statements with instanceof checks

### PR DESCRIPTION
**Pull request by XenoAmess**
as title

### CODE REVIEW

File: spring-test/src/main/java/org/springframework/test/context/web/AbstractGenericWebContextLoader.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ protected void configureWebResources(GenericWebApplicationContext context,
 
 		// If the WebApplicationContext has no parent or the parent is not a WebApplicationContext,
 		// set the current context as the root WebApplicationContext:
-		if (parent == null || (!(parent instanceof WebApplicationContext))) {
+		if ((!(parent instanceof WebApplicationContext))) {
```
Comment (sbrannen) [inline spring-test/src/main/java/org/springframework/test/context/web/AbstractGenericWebContextLoader.java:184]:
```suggestion
		if (!(parent instanceof WebApplicationContext)) {
```

**Review by sbrannen — CHANGES_REQUESTED** [review]
Thanks for the PR.

I've requested a simple change.

File: spring-test/src/main/java/org/springframework/test/context/web/AbstractGenericWebContextLoader.java (hunk 1 above)
Comment (XenoAmess) [reply spring-test/src/main/java/org/springframework/test/context/web/AbstractGenericWebContextLoader.java]: @sbrannen done. and commits squashed.

**Comment by XenoAmess** [comment]
feedback provided above.

**Review by sbrannen — APPROVED** [review]

**Comment by sbrannen** [comment]
This has been merged into `master`.

Thanks

**Pull request merged**